scripts/generic_https_transfer.py

```diff
@@ -65,6 +66,7 @@ def main(presigned_url_file_path: str, filenames: bool):
     for idx, url in enumerate(presigned_urls):
         filename = names[idx] if names else os.path.basename(url).split('?')[0]
         j = batch.new_job(f'URL {idx} ({filename})')
+        j.spot(is_spot=non_preemptible_vm)
```

Comment:
“Spot” means the same as “preemptible”… so with the variable being true for a non-preemptible VM this would need to be `is_spot=not non_preemptible_vm`.